savoirfairelinux/sous-chef
# Fixed #704: Change Member.address from ForeignKey to OneToOneField.

## Fixes #704 by lingxiaoyang

### Changes proposed in this pull request:

* Model change: ForeignKey -> OneToOneField
* Model migration: deals with shared address instances.
* Unit test on model migration.

### Status


### How to verify this change

Before applying the migration, in Django backend admin, make two or more clients linking to the same Address instance.

Do the migration. Then check the address instances of these clients, which should then be separated.

### Deployment notes and migration


### New translatable strings

none

### Additional notes

none
